Extension/src/LanguageServer/client.ts

```diff
@@ -3504,12 +3504,28 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: false
```

Comment:
This false can be replaced by `copy` and the `if` below removed.

Reply:
@browntarik Why is this resolved? I don't see the resolution? Do you have unpushed changes? Usually issues are only resolved after the resolution has been pushed.

Reply:
If you decide to make `copy` as an optional parameter, you can use the nullish coalescing `??` operator to evaluate the value of `copy` to determine what value to assign to `copyToClipboard`.

**Example:**
`copyToClipboard: copy ?? false;`

**Explanation:**
When `copy` is **not** undefined, it will use the value on the left hand side of `??` to assign to `copyToClipboard`. If `copy` evaluates as undefined, it will use `false` the value on the right hand side of `??` as the value to assign to `copyToClipboard`.